Server/src/Server.Domain/Entities/Publisher.cs

```diff
@@ -1,17 +1,13 @@
-﻿using System;
-using System.Collections.Generic;
-using System.ComponentModel.DataAnnotations;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+﻿using System.ComponentModel.DataAnnotations;
+// ReSharper disable CollectionNeverUpdated.Global
 
-namespace Server.Domain.Entities
+#pragma warning disable CS8618
+
+namespace Server.Domain.Entities;
+
+public class Publisher
 {
-    public class Publisher
-    {
-        public Guid Id { get; set; }
-        [Required, MaxLength(50)]
-        public string Name { get; set; }
-        public ICollection<Book> Books { get; set; }
-    }
-}
+    public Guid Id { get; set; }
+    [Required, MaxLength(50)] public string Name { get; set; }
```

Comment:
tu coś dziwnego się zrobiło, dwie linie złączone

Reply:
ta bo to autoformat, mogę poprawić